tests/agent_tests/agent_tests.cc

```diff
@@ -78,3 +78,27 @@ TEST_P(AgentTests, SetThenGetEconParam) {
   EXPECT_EQ(data, -1.0);
 }
 
+TEST_P(AgentTests, PV) {
+  double present_value = agent_->PV(10, 0.05, 10000.0, 100.0);
```

Comment:
Probably makes sense to test with only F, only A, with both, and with i = 0, i != 0, n<0, n=0, n>0